app/src/main/java/com/kickstarter/libs/featureflag/FeatureFlagClient.kt

```diff
@@ -137,7 +137,7 @@ class FeatureFlagClient(
     }
 
     override suspend fun fetchAndActivate(): Boolean {
-        val isInitialized = remoteConfig?.fetchAndActivate()?.await().isTrue()
+        val isInitialized = remoteConfig?.fetchAndActivate()?.await().isNotNull()
 
         log("${this.javaClass} fetchAndActivate completed: $isInitialized")
```

Comment:
nitpick: We can change this log and variable name really, as we discover the other day taking a closer look to the documentation, for `fectchAndActivate`will only be true in those instances where there has been any configuration change applied, for example ff value has changed or new feature flag added.

<img width="757" height="310" alt="Screenshot 2025-09-03 at 12 57 39 PM" src="https://github.com/user-attachments/assets/6c37229c-bd90-455d-860b-5bc94f56942d" />